packages/clis/parser/game-files/sii-schemas.ts

```diff
@@ -541,6 +541,36 @@ export const CitySiiSchema: JSONSchemaType<CitySii> = object(
   [],
 );
 
+export interface MileageTargetsSii {
+  mileageTarget: Record<
+    string,
+    {
+      editorName: string;
+      defaultName: string;
+      names: number | string[];
+      distanceOffset: number;
+      nodeUid: string;
+      position: [number, number, number];
```

Comment:
Because of how the schema is declared as:

```ts
        position: fixedLengthArray(nullable(number), 3),
```

this should be typed as `[number | undefined, number | undefined, number | undefined]` to match.

But I think what we want is:

```
[number, number, number] | [undefined, undefined, undefined]
```

because AFAICT, `position` is only ever fully populated with valid data, or fully populated with the "no data" marker.

See `.diff` for changes related to this.

Reply:
Ahyes.

I ended up with `[number, number, number] | [null, null, null]`, which seems more appropriate for a JSON schema.